src/server/services/dataService/models/phase.js

```diff
@@ -15,6 +15,10 @@ export default function phaseModel(thinky) {
 
       number: number()
         .integer()
+        .min(1)
```

Comment:
Should `.min` be 2, since we only have phases 2, 3, 4?

Reply:
No; that constraint isn't really part of the core nature of this model/type. The fact that we only have these phases is circumstantial, and I'd almost rather remove this constraint altogether than to set a minimum to 2. It's largely irrelevant since the only way this value gets set/changed is through syncing the YAML config files.